### DISCUSSION TITLE
Stuck on the Download Klipper configuration step

### DISCUSSION THREAD

**Discussion by Trikke** [bassamanator/Sovol-SV06-firmware#60] [Q&A] [answered]
I downloaded and installed [OctoKlipper](https://github.com/guysoft/OctoKlipperPi). I can connect to the Pi and Octoprint is running and klipper seems to be installed.

However, I am missing the `~/printer_data/config` folder. These are the only folders that exist in the pi home folder:

```bash
pi@octoprint:~ $ ls
klipper  mjpg-streamer  OctoPrint  oprint  printer.cfg  scripts
``` 

OctoKlipper does mention the `Klipper Config Directory` is just `~/`.

How should i continue with the installation?

**Comment by bassamanator** [comment, marked as answer]
~~I'm not sure I follow: you have OctoKlipper installed on a Rpi which will be hosting Klipper?~~
Never mind, I thought octoklipper was something that you install on a smartphone.

Go through `printer.cfg` and make the appropriate adjustments to make sure the directory structure makes sense with your changes.

Octoprint really isn't need when using Mainsail/Fluidd, and it takes up more resources.

**Reply by Trikke** [reply]
Yeah so i've been reading up and installed Mainsail. Which was all very easy and had the correct folder structure. Copied the git repo and the printer was immediately up and running! Thanks for all the effort you put into this!

**Reply by bassamanator** [reply]
Awesome @Trikke !

Please ⭐star⭐ the repo if you found it useful. Thanks!
